pcsx2/VMManager.cpp

```diff
@@ -1654,6 +1654,23 @@ static void HotkeyCycleSaveSlot(s32 delta)
 	}
 }
 
+static void HotkeyLoadStateSlot(s32 slot)
+{
+	if (s_game_crc == 0)
```

Comment:
Won't this prevent save/load in the bios?

Reply:
why would you need to load a state in the bios?

However a more applicable problem related to this, PS1 games are functionally registered as running the BIOS, so you won't be able to save states in PS1 games.

Reply:
It won't save anyway currently (the file generator returns an empty string).

You can use the manual save to file (in the menu, pick a filename) if you really want to.